standalone/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "standalone"
+version = "0.1.0"
+authors = ["tyfkda <tyfkda@gmail.com>"]
+
+[dependencies]
+sdl2 = "0.32.2"
+rustynes = { path = ".." }
+
+[[bin]]
+name = "rustynes"
```

Comment:
Please add following description.

```
path = "src/main.rs"
required-features = ["binary"]
```

this is because I ran into following warning.

```
warning: /Users/bokuweb/ghq/github.com/bokuweb/rustynes/standalone/Cargo.toml: An explicit [[bin]] section is specified in Cargo.toml which currently
disables Cargo from automatically inferring other binary targets.
This inference behavior will change in the Rust 2018 edition and the following
files will be included as a binary target:

* /Users/bokuweb/ghq/github.com/bokuweb/rustynes/standalone/src/main.rs
```

Reply:
thanks:)I see.please add only path.I'll merge it later!

Reply:
thanks:)I see.please add only path.I'll merge it later!